careerfairsystems/arkad-app
# Camera permissions

## Change description

Adds a redux state to keep track of if the user has accepted giving camera permission. If they click yes on the alert, they won't be prompted again about permission. 

## How to verify

1. Log on as company
2. Click + for Camera and accept both dialogues. 
3. Go back and check that you don't get asked about permissions but go straight to camera when clicking +

## Issues fixed

#285
